doc-content/enterprise-only/installation/kie-server-thread-efficiency.adoc

```diff
@@ -0,0 +1,18 @@
+[id='kie-server-thread-efficiency-con_{context}']
+= Thread efficiency
+
+To ensure that the optimal number of threads are used, set the value of the threading system properties to the sum of the number of CPUs plus one.
+
+. In your Red Hat Process Automation Manager installation directory, navigate to the `standalone-full.xml` file. For example, if you use a {EAP}  installation for {PRODUCT}, go to `$EAP_HOME/standalone/configuration/standalone-full.xml`.
```

Comment:
Red Hat Process Automation Manager, change to {PRODUCT}. If you don't this will be wrong for the Decision Manager doc.

Reply:
D'oh.